pkg/ccl/backupccl/backup_planning.go

```diff
@@ -718,6 +732,13 @@ func backupPlanHook(
 			return err
 		}
 
+		// Check that a node will currently be able to run this before we create it.
+		if coordinatorLocality.NonEmpty() {
+			if _, err := p.DistSQLPlanner().GetAllInstancesByLocality(ctx, coordinatorLocality); err != nil {
+				return err
```

Comment:
What does this error end up looking like? I'm wondering if we should wrap it, mentioning the COORDINATOR_LOCALITY option so the user knows what they might need to change.

Reply:
it looks like `pq: no instances found matching locality filter %s`. It includes the filter that didn't match, which seems more useful than the field they put that filter in?